.bcr/presubmit.yml

```diff
@@ -0,0 +1,15 @@
+matrix:
+  platform:
+    - macos_arm64
+    - ubuntu2404
+    - windows
```

Comment:
Could you look into what has been going wrong in the Windows case?
https://github.com/dtolnay/cxx/actions/runs/13841971991/job/38731434026

> `ERROR: C:/users/runneradmin/_bazel_runneradmin/dzh43mlk/external/rules_rust+/util/process_wrapper/BUILD.bazel:31:36: Compiling Rust (without process_wrapper) bin process_wrapper (6 files) [for tool] failed: (Exit 1): bootstrap_process_wrapper.bat failed: error executing Rustc command (from target @@rules_rust+//util/process_wrapper:process_wrapper)`

> `LINK : fatal error LNK1181: cannot open input file 'bazel-out\x64_windows-opt-exec-ST-d57f47055a04\bin\external\rules_rust++rust+rw-1704990954_tools\rust_toolchain\lib\rustlib\x86_64-pc-windows-msvc\lib\librustc_std_workspace_alloc-2cc5ffc24865e22c.rlib'`

Reply:
I opened https://github.com/dtolnay/cxx/pull/1468 for this

Reply:
I would also be happy to drop windows for now until the underlying issues are resolved. This doesn't prevent anyone from pulling the module on windows, just means windows won't be tested for releases to BCR.
```suggestion
```